WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -310,6 +315,31 @@ private extension StorePickerViewController {
         return WordPressAuthenticator.shared.unifiedStyle?.viewControllerBackgroundColor ?? .listBackground
     }
 
+    func presentActionMenu(from sourceView: UIView) {
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.view.tintColor = .text
+        let logOutAction = UIAlertAction(title: Localization.ActionMenu.logOut, style: .default) { [weak self] _ in
+            self?.restartAuthentication()
```

Comment:
Nit: Do we want to track this event? Perhaps in a future PR.

Reply:
We haven't been tracking this action, but sounds like a good idea. I added it to the analytics plan https://github.com/woocommerce/woocommerce-ios/issues/8248 pe5sF9-I3-p2#comment-1330.